rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -136,9 +137,9 @@ class AsyncParametersClient
   std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>
   set_parameters(
     std::vector<rclcpp::parameter::ParameterVariant> parameters,
-    std::function<
-      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)
-    > callback = nullptr)
+    std::function<void(
+      std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)> callback =
+    nullptr)
```

Comment:
I changed this line to look like before, ran uncrustify again and submitted the changes. Uncrustify still changed the style, though.